api/types/src/convert.rs

```diff
@@ -372,7 +371,7 @@ impl<'a, R: MoveResolver + ?Sized> MoveConverter<'a, R> {
                 let MoveScriptBytecode { bytecode, abi } = code.try_parse_abi();
                 match abi {
                     Some(func) => {
-                        let args = self.try_into_move_values(func, arguments)?;
+                        let args = self.try_into_vm_values(func, arguments)?;
```

Comment:
why rename move to vm?

Reply:
introducing `try_into_move_value()` later in the stack, where it means the api `MoveValue` type

Reply:
what's the difference between the two?

Reply:
`api::types::MoveValue` which is a JSON conversion helper class and `move_core_types::value::MoveValue`.

Most of the `try_into_*` methods here yields API helper classes.